File: tt-api/src/Domain/Entities/CustomerClient.cs
using Domain.ValueObjects;

namespace Domain.Entities;

public class CustomerClient
{
    public string Code { get; set; } = null!;
    public string CompanyCode { get; set; } = null!;
    public string CustomerCode { get; set; } = null!;
    public string Name { get; set; } = null!;
    public string PrimaryAddress { get; set; } = null!;
    public string BillingAddress { get; set; } = null!;
    public string ShippingAddress { get; set; } = null!;
    public string Pic1 { get; set; } = null!;
    public string Pic2 { get; set; } = null!;
    public CustomerClientStatus Status { get; set; } = null!;
    public string CreatedBy { get; set; } = null!;
    public DateTime CreatedDate { get; set; }
    public string? CancelledBy { get; set; }
    public DateTime? CancelledDate { get; set; }
    public string? RevisedBy { get; set; }
    public DateTime? RevisedDate { get; set; }
}

File: ttlogix-web/App/ClientApp/src/store/models/customerClient.ts
export class CustomerClient {
    code: string;
    name: string;
    contactPerson: string;
    telephoneNo: string;
    faxNo: string;
    status: number;

    customerCode: string;
    companyCode: string;
    primaryAddress: string | null;
    billingAddress: string | null;
    shippingAddress: string | null;
    pic1: string | null;
    pic2: string | null;
}